pkg/decode/decode_protobuf_test.go

```diff
@@ -7,375 +7,11 @@ import (
-			flow: &pbflow.Record{
-				Interface:     "eth0",
-				EthProtocol:   2048,
-				Bytes:         456,
-				Packets:       123,
-				Direction:     pbflow.Direction_EGRESS,
-				TimeFlowStart: timestamppb.New(someTime),
-				TimeFlowEnd:   timestamppb.New(someTime),
-				Network: &pbflow.Network{
-					SrcAddr: &pbflow.IP{
-						IpFamily: &pbflow.IP_Ipv4{Ipv4: 0x01020304},
-					},
-					DstAddr: &pbflow.IP{
-						IpFamily: &pbflow.IP_Ipv4{Ipv4: 0x05060708},
-					},
-					Dscp: 64,
-				},
-				DataLink: &pbflow.DataLink{
-					DstMac: 0x112233445566,
-					SrcMac: 0x010203040506,
-				},
-				Transport: &pbflow.Transport{
-					Protocol: 6,
-					SrcPort:  23000,
-					DstPort:  443,
-				},
-				AgentIp: &pbflow.IP{
-					IpFamily: &pbflow.IP_Ipv4{Ipv4: 0x0a090807},
-				},
-				Flags:                  0x100,
-				PktDropBytes:           100,
-				PktDropPackets:         10,
-				PktDropLatestFlags:     0x200,
-				PktDropLatestState:     6,
-				PktDropLatestDropCause: 5,
-				DnsLatency:             durationpb.New(someDuration),
-				DnsId:                  1,
-				DnsFlags:               0x8001,
-				DnsErrno:               0,
-				TimeFlowRtt:            durationpb.New(someDuration),
-			},
-			expected: &config.GenericMap{
-				"FlowDirection":          1,
-				"Bytes":                  uint64(456),
-				"SrcAddr":                "1.2.3.4",
-				"DstAddr":                "5.6.7.8",
-				"Dscp":                   uint32(64),
-				"DstMac":                 "11:22:33:44:55:66",
-				"SrcMac":                 "01:02:03:04:05:06",
-				"Duplicate":              false,
-				"Etype":                  uint32(2048),
-				"Packets":                uint64(123),
-				"Proto":                  uint32(6),
-				"SrcPort":                uint32(23000),
-				"DstPort":                uint32(443),
-				"Flags":                  uint32(0x100),
-				"TimeFlowStartMs":        someTime.UnixMilli(),
-				"TimeFlowEndMs":          someTime.UnixMilli(),
-				"Interface":              "eth0",
-				"AgentIP":                "10.9.8.7",
-				"PktDropBytes":           uint64(100),
-				"PktDropPackets":         uint64(10),
-				"PktDropLatestFlags":     uint32(0x200),
-				"PktDropLatestState":     "TCP_CLOSE",
-				"PktDropLatestDropCause": "SKB_DROP_REASON_TCP_CSUM",
-				"DnsLatencyMs":           someDuration.Milliseconds(),
-				"DnsId":                  uint32(1),
-				"DnsFlags":               uint32(0x8001),
-				"DnsFlagsResponseCode":   "FormErr",
-				"DnsErrno":               uint32(0),
-				"TimeFlowRttNs":          someDuration.Nanoseconds(),
-			},
-		},
-	}
-
-	for _, tt := range tests {
-		rawPB, err := proto.Marshal(tt.flow)
-		require.NoError(t, err)
-		out, err := decoder.Decode(rawPB)
-		require.NoError(t, err)
-		assert.NotZero(t, out["TimeReceived"])
-		delete(out, "TimeReceived")
-		assert.Equalf(t, *tt.expected, out, tt.name)
-	}
-}
+// NOTE: more tests in convert_flp_test.go
```

Comment:
did intentionally deleted conversion from gRPC to generic map test cases here ?

Reply:
yes, like I wrote this test was moved (and sightly changed) to convert_flp_test, but I admit the name is misleading, I'll rename that file to clarify it's testing BOTH grpc and direct-flp conversions